app/commands/v2/put_content.rb

```diff
@@ -98,6 +99,17 @@ def create_links(edition)
         end
       end
 
+      def create_content_id_alias
+        content_id_alias = payload.dig(:details, :content_id_alias)
+        return unless content_id_alias && EmbeddedContentFinderService::SUPPORTED_DOCUMENT_TYPES.include?(payload[:document_type])
+
+        if ContentIdAlias.find_by(name: content_id_alias)
```

Comment:
Will this fail if I publish more than one edition for the same document? This reads like we're expecting editions to have unique aliases, which I don't think is right.

Reply:
Yeah I think we should only error if the content ID differs:
- if the alias and content ID match an existing `ContentIdAlias`, do nothing
- if the alias matches an existing `ContentIdAlias`, error
- if the alias doesn't exist, create one for the given content ID